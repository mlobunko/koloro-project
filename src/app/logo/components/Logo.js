import React from "react";

import { StyledLogoO, StyledLogoWords, StyledReactTilt } from "./styles";

export const Logo = ({ landscape }) => (
  <React.Fragment>
    <StyledLogoWords landscape={landscape} />
    <StyledReactTilt
      landscape={landscape}
      options={{
        "position-base": "window",
        max: 80,
        reset: true,
        reverse: true,
        perspective: 500
      }}
      className="tilt-elem my-tilt"
    >
      <StyledLogoO landscape={landscape} className="Tilt-inner" />
    </StyledReactTilt>
  </React.Fragment>
);
Logo.displayName = "Logo";

export default Logo;
