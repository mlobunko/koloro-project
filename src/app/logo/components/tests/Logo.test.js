import React from "react";
import { shallow } from "enzyme";

import { Logo } from "../Logo";

describe("render", () => {
  it("renders correctly", () => {
    const landscape = true;
    const wrapper = shallow(<Logo landscape={landscape} />);
    expect(wrapper).toMatchSnapshot();
  });
});
