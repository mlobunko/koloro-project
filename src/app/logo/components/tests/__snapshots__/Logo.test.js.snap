// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<Fragment>
  <StyledLogoWords
    landscape={true}
  />
  <Styled(t)
    className="tilt-elem my-tilt"
    landscape={true}
    options={
      Object {
        "max": 80,
        "mobile": false,
        "perspective": 500,
        "position-base": "window",
        "reset": true,
        "reverse": true,
      }
    }
  >
    <StyledLogoO
      className="Tilt-inner"
      landscape={true}
    />
  </Styled(t)>
</Fragment>
`;
